src/twisted/python/_pydoctor.py

```diff
@@ -114,9 +115,13 @@ def _getURLAsHEAD(self, url):
         @type url: L{str}
 
         @return: The response for the HEAD method.
-        @rtype: urllib2 response
+        @rtype: urllib2 response or L{None}
         """
-        return urllib2.urlopen(HeadRequest(url))
+        try:
+            return urllib2.urlopen(HeadRequest(url))
+        except Exception as e:
+            print("Error opening {}: {}".format(url, e))
```

Comment:
The link you provided just takes me back to this diff.  Are you referring to this docstring: https://github.com/twisted/twisted/blob/trunk/src/twisted/python/_pydoctor.py#L38